api/auth/signup.js

```diff
@@ -18,6 +16,7 @@ for(let i =0; i < database.users.length; i++){
     })
   }
 }
+// if not , add new user to the end of the database array
 if(!match){
   database.users.push({
     firstname:firstname,
```

Comment:
The best practice is to encrypt the users password using a secret key and `bcrypt`, when saving the password. When authenticating, you can compare the encrypted password with the one the user provides.